src/libsync/discovery.h

```diff
@@ -88,6 +88,16 @@ class ProcessDirectoryJob : public QObject
     /** Start up to nbJobs, return the number of job started; emit finished() when done */
     int processSubJobs(int nbJobs);
 
+    void setIsInsideEncryptedTree(bool isInsideEncryptedTree)
```

Comment:
My apologies, should have been clearer on the Qt convention:
```
void setValue(int value);
int value() const;
```
But for bools it reads like this:
```
void setCondition(bool condition);
bool isCondition() const;
```

So you'll have to drop the `Is` part in that setter name.

Reply:
@er-vin Ok, I thought your emphasis was on adding **is**.

Reply:
Yes, that's why I realized I was not thorough enough earlier. Sorry again.